core/src/main/java/org/mule/DefaultMuleEvent.java

```diff
@@ -69,6 +70,8 @@
 
     private static Log logger = LogFactory.getLog(DefaultMuleEvent.class);
 
+    public static final int DEFAULT_LENGTH_MULE_EVENT_ID = UUID_LENGTH + 2 ;
```

Comment:
because In DefaultMuleEvent.generateEventId() -> DefaultMuleContext.getUniqueIdString(), UUID is concatenated with default clusterNodeId (zero) and "-" character.